### ISSUE TITLE
continuousVertical with Fixed Background and autoScrolling off

### ISSUE THREAD

**Issue by JakeHenshall** [alvarotrigo/fullPage.js#3863]
### Description

Is there any way to use continuousVertical with Fixed Background and autoScrolling off to keep loading in the slides to create something similar to this site -> https://madebyarticle.com/

Is there any callback functions that I could use to keep adding the next slide?

### Link to isolated reproduction with no external CSS / JS
[Ideally in jsfiddle.net (https://jsfiddle.net/alvarotrigo/ea17skjr) or codepen.io (https://codepen.io/alvarotrigo/pen/NxyPPp), links to personal websites won't be reviewed unless isolated. Reported issues without a reproduction might get closed.]

https://codepen.io/JakeHenshall/pen/NWPBawv

### Steps to reproduce it
1. [First step]
2. [Second step]
3. [and so on...]

### Versions 
[Browser, operating system, device, ...]
Chrome / Mac OS Catalina

**Comment by alvarotrigo** [comment]
> Is there any way to use continuousVertical with Fixed Background and autoScrolling off

Nope! Unfortunately continuous vertical only works with autoScrolling at the moment. 

I'll keep this issue open as a possible future enhancement! 👍

**Comment by JakeHenshall** [comment]
@alvarotrigo That sounds great.
